Index: ../moie-core/moie-core.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../moie-core/moie-core.iml b/../moie-core/moie-core.iml
new file mode 100644
--- /dev/null	(date 1618784462878)
+++ b/../moie-core/moie-core.iml	(date 1618784462878)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: ../moie-core/ormconfig.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\n   \"type\": \"mysql\",\n   \"host\": \"localhost\",\n   \"port\": 8889,\n   \"username\": \"root\",\n   \"password\": \"root\",\n   \"database\": \"moie-lucy-v2\",\n   \"synchronize\": true,\n   \"logging\": false,\n   \"entities\": [\n      \"src/entity/**/*.ts\"\n   ],\n   \"migrations\": [\n      \"src/migration/**/*.ts\"\n   ],\n   \"subscribers\": [\n      \"src/subscriber/**/*.ts\"\n   ],\n   \"seeds\": [\n      \"src/seeds/*.ts\"\n   ],\n   \"cli\": {\n      \"entitiesDir\": \"src/models\",\n      \"migrationsDir\": \"src/migration\",\n      \"subscribersDir\": \"src/subscriber\"\n   }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../moie-core/ormconfig.json b/../moie-core/ormconfig.json
--- a/../moie-core/ormconfig.json	(revision a3c61c903bc73cfa0961b5e03123245a3f38a95c)
+++ b/../moie-core/ormconfig.json	(date 1618784462770)
@@ -1,7 +1,7 @@
 {
    "type": "mysql",
    "host": "localhost",
-   "port": 8889,
+   "port": 3306,
    "username": "root",
    "password": "root",
    "database": "moie-lucy-v2",
